### PR TITLE
Add OS dependencies for installing requests_kerberos completely

### PR DESCRIPTION
**Pull request by akanshi-elastic**
## Closes https://github.com/elastic/connectors-python/issues/1310###
This PR include:
1. changes in Makefile to add os dependency to install requests_kerberos module without any errors.

#### Pre-Review Checklist
- [x] this PR has a meaningful title
- [x] this PR links to all relevant github issues that it fixes or partially addresses
- [x] this PR has a thorough description
- [x] Tested the changes locally
- [x] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [x] Considered corresponding documentation changes

### CODE REVIEW

**Comment by akanshi-elastic** [comment]
We also need to add `make pre-install` command in readme file.

**Review by artem-shelkovnikov — COMMENTED** [review]
Why have it as a separate `pre-install` command instead of having it in `install`?

File: Makefile [hunk 1]
```diff
@@ -11,6 +11,10 @@ bin/python:
 	$(PYTHON) -m venv .
 	bin/pip install --upgrade pip
 
+pre-install: bin/python
+	bin/pip install -r requirements/pre_install.txt
+	bin/python install.py develop
```
Comment (artem-shelkovnikov) [inline Makefile:16]: Why `develop`? This is supposed to be running on production systems as well.

**Comment by akanshi-elastic** [comment]
Build is failing but I guess the reason is that pre_install.txt is not already added in connectors-python. Let me know if anything needs to be changed from our side to pass the build

File: requirements/pre_install.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pre_install.txt
```
[the remark this answers is not in the document]
Comment (akanshi-elastic) [reply requirements/pre_install.txt:1]: Oh my bad. I just missed see this. let me fix it

**Comment by seanstory** [comment]
Closing, as this is super stale

**Pull request closed without merging**